### PR TITLE
[Security] Added Ukrainian translations

### LINKED ISSUE #41088: Missing translations for Ukrainian (uk)
Hello,

There are some missing translation for Ukrainian (uk). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by andrii-bodnar**
| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #41088 
| License       | MIT
| Doc PR        | 

Added some missing Ukrainian translations.

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Посилання для входу недійсне, або термін його дії закінчився.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Забагато невдалих спроб входу. Будь ласка, спробуйте через %minutes% хвилину.</target>
```
Comment (Comrade42) [inline src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf]:
```suggestion
                <target>Забагато невдалих спроб входу. Будь ласка, спробуйте знову через %minutes% хвилину.</target>
```

File: src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf [hunk 2]
```diff
@@ -70,6 +70,14 @@
...
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Забагато невдалих спроб входу. Будь ласка, спробуйте через %minutes% хв.</target>
```
Comment (Comrade42) [inline src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf]:
```suggestion
                <target>Забагато невдалих спроб входу. Будь ласка, спробуйте знову через %minutes% хвилин.</target>
```

**Review by Comrade42 — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf (hunk 2 above)
Comment (andrii-bodnar) [reply src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf]: @Comrade42, I used "хв." here, due to a possible situation of incorrect translation at some values of `%minutes%`.

For example:
`%minutes%` = 10: should be `... спробуйте через 10 хвилин.`
`%minutes%` = 21: should be `... спробуйте через 21 хвилину.`

`хв.` will be correct in both cases. Also, for Russian translations, the same approach is used.

Comment (Comrade42) [reply src/Symfony/Component/Security/Core/Resources/translations/security.uk.xlf]: Good, it sounds reasonable to me.

**Comment by andrii-bodnar** [comment]
@Comrade42 I've addressed all the comments. Thanks for the review.

Status: Needs Review

**Comment by Comrade42** [comment]
Looks good to me, thank you 👍

**Comment by andrii-bodnar** [comment]
@chalasr @wouterj please review this PR :slightly_smiling_face:

**Review by chalasr — APPROVED** [review]

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**